public/components/query_compare/search_result/index.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiPageHeaderSection,
+  EuiCode,
+  EuiPanel,
+  EuiSpacer,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiSplitPanel,
+  EuiText,
+} from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+
+import { CoreStart, ChromeBreadcrumb } from '../../../../../../src/core/public';
+import { SearchConfigsPanel } from './search_components/search_configs';
+import { SearchInputBar } from './search_components/search_bar';
+import { SearchResultTable } from './result_components/search_result_table';
+import { ServiceEndpoints } from '../../../constants';
+
+const defaultQuery = JSON.stringify({
+  query: {
+    term: { title: '%searchInput%' },
```

Comment:
I think the UI mocks changed `%searchInput%` to `%SearchQuery%`. Let's reflect the same here.